x/interchainqueries/types/tx_test.go

```diff
@@ -345,6 +403,18 @@ func TestMsgUpdateQueryRequestValidate(t *testing.T) {
 			},
 			sdkerrors.ErrInvalidRequest,
 		},
+		{
+			"too many keys",
+			func() sdktypes.Msg {
+				return &iqtypes.MsgUpdateInterchainQueryRequest{
+					QueryId:         1,
+					NewKeys:         craftKVKeys(200),
+					NewUpdatePeriod: 0,
+					Sender:          TestAddress,
+				}
+			},
+			iqtypes.ErrTooManyKVQueryKeys,
+		},
```

Comment:
Please expand the update test according to the latest changes in terms of update payload validation